infra/gcp/clusters/modules/k8s-infra-gke-project/main.tf

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2020 The Kubernetes Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/// Create the project in which we're creating the cluster
+resource "google_project" "project" {
+  name            = var.project_name
+  project_id      = var.project_name
+  org_id          = "758905017065"          // kubernetes.io
```

Comment:
Could be a `string` variable with a default value.

Reply:
I started with that, but moved to hardcodes because I don't think we want to give people a choice to use other orgs/billing (in much the same way our `ensure_project` bash sets these for everything)